app/app.go

```diff
@@ -215,17 +229,6 @@ func (a *app) Start() error {
 	// update our public IP, [p.config.IPUdater] is a no-op implementation.
 	go a.config.IPUpdater.Dispatch(log)
 
-	if err := a.node.Initialize(&a.config, log, logFactory); err != nil {
-		log.Fatal("error initializing node",
-			zap.Error(err),
-		)
-		mapper.UnmapAllPorts()
-		a.config.IPUpdater.Stop()
```

Comment:
@joshua-kim are these two cleanups lost in refactoring?

Reply:
These are closed in L241 now - previously we had to close them if initialization failed since otherwise the goroutines would be hanging.